### DISCUSSION TITLE
blog/cheating-in-splatoon

### DISCUSSION THREAD

**Discussion by giscus** [NotNite/giscus#1] [Announcements]
# blog/cheating-in-splatoon

https://notnite.com/blog/cheating-in-splatoon

**Comment by redcathode** [comment]
> I didn’t have a soldering iron upstairs, so I just kind of touched the wire to the right pin using the power of friendship  

no!!! use a socket-to-socket jumper smh my head
<details>
<summary>                                          </summary>
/j /lh /lh  
<br>
also definitely a necropost at this point lol
</details>
